.github/workflows/main.yml

```diff
@@ -141,30 +141,43 @@ jobs:
         timeout-minutes: 6
         run: yarn test:fastboot ${{ matrix.scenario }}
 
-  basic-tests-ie11:
+  browser-tests:
     needs: [lint]
+    timeout-minutes: 20
     strategy:
+      fail-fast: false
       matrix:
         scenario: [with-native-fetch, with-ember-fetch-no-jquery, with-ember-fetch-and-jquery, with-jquery]
-    runs-on: windows-latest
-    env:
-      TEST_IE11: true
-      TARGET_IE11: true
+        launcher: [Chrome, Firefox, IE]
+        os: [macos-latest, windows-latest, ubuntu-latest]
+        exclude:
+          - os: macos-latest
+            launcher: IE
+          - os: ubuntu-latest
+            launcher: IE
+          - os: ubuntu-latest
+            launcher: Firefox
+          - os: windows-latest
+            launcher: Firefox
```

Comment:
<img width="369" alt="Screen Shot 2020-07-23 at 8 25 01 AM" src="https://user-images.githubusercontent.com/7374640/88312792-6eeff800-ccc7-11ea-89dd-17b4ff04ea60.png">

Tests outside of macos seemed flakey.  I did see them pass or twice, but more often than not caused a failure in windows and ubuntu.  Would hate to introduce test flakiness.